pkg/morph/deploy/notary.go

```diff
@@ -125,7 +125,15 @@ func enableNotary(ctx context.Context, prm enableNotaryPrm) error {
 // initDesignateNotaryRoleToLocalAccountTick returns a function that preserves
 // context of the Notary role designation to the local account between calls.
 func initDesignateNotaryRoleToLocalAccountTick(ctx context.Context, prm enableNotaryPrm) (func(), error) {
-	localActor, err := actor.NewSimple(prm.blockchain, prm.localAcc)
+	committeeMultiSigM := smartcontract.GetMajorityHonestNodeCount(len(prm.committee))
+	committeeMultiSigAcc := wallet.NewAccountFromPrivateKey(prm.localAcc.PrivateKey())
+
+	err := committeeMultiSigAcc.ConvertMultisig(committeeMultiSigM, prm.committee)
```

Comment:
https://pkg.go.dev/github.com/nspcc-dev/neo-go/pkg/smartcontract#CreateMajorityMultiSigRedeemScript

Reply:
how should i construct `localActor` then?